docs/api/platform/Error.md

```diff
@@ -298,12 +329,99 @@ void save_all_errors() {
     mbed_clear_all_errors();
 }
 ```
+
+#### Using `mbed_get_reboot_error_info()` to retrieve the reboot error info
+The error context captured by the error handling system can be retrieved using mbed_get_reboot_error_info() API. See the below code for example usage of that API. 
```

Comment:
Open to alternative phrasing here, mainly trying to remove the API term if this sentence is only referencing a single function call (could have the usage of API in a sentence wrong as well, let me know if this is fine as is).

```suggestion
The error context captured by the error handling system can be retrieved using `mbed_get_reboot_error_info()`. See the below code for an example using that function. 
```